staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5073,6 +5073,9 @@ type NodeStatus struct {
 	// Note: This field is declared as mergeable, but the merge key is not sufficiently
 	// unique, which can cause data corruption when it is merged. Callers should instead
 	// use a full-replacement patch. See https://pr.k8s.io/79391 for an example.
+	// Besides, it is possible its addresses get changed because the Node or kubelet
```

Comment:
What do you think about something like this:
```
Consumers should assume that a addresses can change during the
lifetime of a Node. However, there a some exceptions where this may not
be possible, such as Pods that inherit a node's address in its own status or
consumers of the downward API (status.hostIP).
```

Reply:
Yes, we can use this.